src/vaadin-date-time-picker.html

```diff
@@ -179,13 +178,17 @@
              */
             required: {
               type: Boolean,
+              reflectToAttribute: true,
               value: false
             },
```

Comment:
Is this `reflectToAttribute` needed for styling purposes or what? I noticed that vaadin-text-field does this too, but date-picker does not reflect it so I'm not sure if it should have it here or not.

Reply:
This was done for testing purposes and pushed to remote to save progress (PR is still `in progress`). Removed now.